### PR TITLE
updated readme with steps to install RabbitMQ server

### PR DESCRIPTION
**Pull request by yash1337**

### CODE REVIEW

**Comment by nathankurt** [comment]
Don't know how i feel about the nbsp in the readmes. It seems pretty hacky and when you clone the file, it makes it less readable from a text editor point of view.

File: README.md [hunk 1]
```diff
@@ -10,10 +10,17 @@ Steps to run a local copy of SpartaHackAPI
...
-3. Create a virtualenv using `mkvirtualenv` using the virtualenvwrapper
-4. Switch the python interpreter to the virtualenv using `workon virtualenv_name` 
-5. Install the dependencies using `pip install -r requirements.txt`
+2. Clone the API and cd into it  
+3. Create a virtualenv using `mkvirtualenv` using the virtualenvwrapper  
+4. Switch the python interpreter to the virtualenv using `workon virtualenv_name`  
+5. Install the dependencies using `pip install -r requirements.txt`  
+6. Install RabbitMQ server using `sudo apt install rabbitmq-server`  
+7. With RabbitMQ server running (it will after first install otherwise run it using `sudo rabbitmq-server`)  
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`sudo rabbitmqctl add_user dev mypassword`
```
Comment (nathankurt) [inline README.md]: This looks pretty bad. Is there a better way to hack it. or list within list?

Comment (yash1337) [reply README.md]: Let me see.

**Comment by yash1337** [comment]
Ahh I see. You just need to use pip3 until you activate the virtualenv after that pip assumes the pip you used to make the virtualenv. I can change pip to pip3 until virtualenv activation.

**Pull request merged**